freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -70,15 +68,24 @@ def handle(self, event):
             self.log_info(msg)
             return []
 
-        # Get original nvrs of all builds in the advisory
-        original_nvrs = [build.original_nvr for build in db_event.builds]
+        # Get builds NVRs from the advisory attached to the message/event and
+        # then get original NVR for every build
+        original_nvrs = set()
+        for product_info in event.advisory.builds.values():
+            for build in product_info['builds']:
+                # Search for the first build that triggered the chain of rebuilds
+                # for every shipped NVR to get original NVR from it
+                original_nvr = ArtifactBuild.get_original_nvr_by_rebuilt_nvr(
+                    db.session, build['nvr'])
+                original_nvrs.add(original_nvr)
```

Comment:
What if `original_nvr` is `None`? This can happen if someone attached a non-Freshmaker build to a BOTAS advisory. It's an unsupported behavior but we've seen it happen.

Reply:
Yes you are right, I should check for None.